prow/cluster/jobs/istio-releases/pipeline/istio-releases.pipeline.release-1.1.yaml

```diff
@@ -10,7 +10,7 @@ istio_rel_pipeline_spec: &istio_rel_pipeline_spec
     testing: build-pool
 
 istio_rel_pipeline_container: &istio_rel_pipeline_container
-  image: gcr.io/istio-testing/istio-builder:v20190819-7f5bc561
+  image: gcr.io/istio-testing/istio-builder:v20190823-25f7c637
```

Comment:
i'm not sure how wise it is to change the 1.1 toolchain.

Reply:
This is intentional. We have already upgraded 1.1 to go 1.12.5, this just bumps from 1.12.9 which should be safe (patch release). This is recommended by 1.1 release manager @duderino